packages/app-project/src/shared/components/SubjectPicker/helpers/searchParams.js

```diff
@@ -0,0 +1,10 @@
+export default function searchParams(data) {
+  let query = ''
+  Object.entries(data).forEach(([key, value]) => {
+    if (value !== '') {
+      query += `@${key}:${value}*`
```

Comment:
⚠️❓ Is there any chance that somebody will try to search for a value that'll break this query? e.g. `title = 'A Concise Guide to Colons: Volume XI: Deluxe Edition: This Time It's Personal'` and `author = 'shaun@zooniverse'`

Reply:
Good point. I think this will also be fixed by the move to Datasette, which uses parameterised SQL queries. Queries against the new API will look something like `/subjects/92752?_sort=priority&Date__contains=December`.

Reply:
I've got a branch set up now which gets this working with Datasette. https://github.com/zooniverse/front-end-monorepo/tree/subject-set-search-api